app/src/main/java/com/github/se/studybuddies/ui/chat/ChatScreen.kt

```diff
@@ -927,6 +990,78 @@ fun FilePickerBox(
       }
 }
 
+@Composable
+fun SendPollMessage(messageViewModel: MessageViewModel, showAddPoll: MutableState<Boolean>) {
+  val question = remember { mutableStateOf("") }
+  val options = remember { mutableStateOf(listOf("")) }
+  val singleChoice = remember { mutableStateOf(true) }
+
+  ShowAlertDialog(
+      modifier = Modifier.testTag("add_poll_dialog"),
+      showDialog = showAddPoll,
+      onDismiss = { showAddPoll.value = false },
+      title = {},
+      content = {
+        Column {
+          OutlinedTextField(
+              label = { Text(stringResource(R.string.poll_question)) },
+              value = question.value,
+              onValueChange = { question.value = it },
+              modifier = Modifier.fillMaxWidth().testTag("add_poll_question_text_field"),
+              textStyle = TextStyle(color = Black),
+              singleLine = true,
+              placeholder = { Text(stringResource(R.string.enter_poll_question)) },
+          )
+          Spacer(modifier = Modifier.height(16.dp))
+
+          Text(text = stringResource(R.string.poll_options))
+
+          LazyColumn(modifier = Modifier.fillMaxWidth().heightIn(max = 200.dp)) {
+            items(options.value) { option ->
+              OutlinedTextField(
+                  value = option,
+                  onValueChange = {
+                    options.value =
+                        options.value.toMutableList().apply {
+                          set(indexOf(option), it)
+                          removeAll { option -> option.isBlank() }
+                          if (lastOrNull()?.isNotBlank() == true) add("")
+                        }
+                  },
+                  modifier = Modifier.fillMaxWidth().testTag("add_poll_options_text_field"),
+                  textStyle = TextStyle(color = Black),
+                  singleLine = true,
+                  placeholder = { Text(stringResource(R.string.enter_poll_options)) },
+              )
+              Spacer(modifier = Modifier.height(8.dp))
+            }
+          }
+          Row(
+              modifier =
+                  Modifier.fillMaxWidth().padding(8.dp).testTag("add_poll_single_choice_row"),
+              verticalAlignment = Alignment.CenterVertically,
+              horizontalArrangement = Arrangement.SpaceBetween) {
+                Text(
+                    text = stringResource(R.string.single_choice),
+                    style = TextStyle(color = Black),
+                    modifier = Modifier.testTag("add_poll_single_choice_text"))
+                Switch(checked = singleChoice.value, onCheckedChange = { singleChoice.value = it })
```

Comment:
Could you change the default colors? For coherence you can re-use the ones already set in TopicScreen. Because for now, when the switch is on, the entire track is blue and the thumb is not visible, which doesnt make it very clear that this is a switch button, and in dark mode, the unchecked switch looks kinda weird

Reply:
True, it was not very well display. Thanks for the indication where to find the solution!